lib/diagnostic.js

```diff
@@ -38,19 +49,35 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let vocalize = function (sound) {
+  console.log(sound);
+}
+
+let pet = {
+  name: 'Mirah',
+  species: 'Canis Felis',
+  breed: 'Common Housecat',
+  vocalization: vocalize(),
```

Comment:
vocalize is a property of pet so
```js
vocalize: function() {
  return this.vocalization;
}
```
and vocalization would just be a string

`vocalization: 'Meow',`